airflow/www/templates/appbuilder/baselayout.html

```diff
@@ -13,7 +32,6 @@
   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   See the License for the specific language governing permissions and
   limitations under the License.
-
 #}
 {% extends 'appbuilder/init.html' %}
```

Comment:
```suggestion

{% extends 'appbuilder/init.html' %}
```
To maintain consistency with other files.